include/ebpf_xdp_test_hooks.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) eBPF for Windows contributors
+// SPDX-License-Identifier: MIT
+#pragma once
+#include <stdint.h>
+
+// This file contains APIs for hooks and helpers that are
+// exposed by netebpfext.sys for use by eBPF xdp test programs.
+
+// XDP_TEST hook.
+typedef struct xdp_test_md_
```

Comment:
I would not rename the type. Keep using the old type name but move in an internal header that only the test code has access to.

Reply:
Please do not create or retain xdp struct name collisions - even if it compiles, it will create confusion for internal and broader community developers if we have definitions spanning repos. The "xdp_test" naming style seems appropriate to me.

Reply:
XDP already has its own automated tests - feel free to discard the eBPF test code entirely.